README.md

```diff
@@ -36,7 +36,12 @@ Lita.configure do |config|
   config.robot.adapter = :slack
   config.robot.admins = ["U012A3BCD"]
   config.adapters.slack.token = "abcd-1234567890-hWYd21AmMH2UHAkx29vb5c1Y"
-  config.adapters.slack.parse = "full" # See https://api.slack.com/docs/formatting#parsing_modes
+
+  # Slack parsing mode options, See https://api.slack.com/docs/formatting#parsing_modes
+  # The following is the recommended configuration and will become the default in the next major version
```

Comment:
These future defaults are probably a good choice, but since it's require for the rest of the functionality in this PR, let's remove the mention of any future defaults and make that decision separately later.